website/src/content/blog/2025-03-31-typespec-1-0-release/typespec_1_0_release.md

```diff
@@ -0,0 +1,215 @@
+---
+slug: 2025-03-31-typespec-1-0-release
+title: "TypeSpec 1.0-RC: Write your API once, let TypeSpec do the rest"
+image: ./TypeSpec_1_0_RC.png
+description: "The TypeSpec 1.0 Release Candidate is here! This open-source project, created by Microsoft, brings a powerful yet concise API modeling language that generates server code, client libraries, schemas, and documentation from a single source of truth."
+publishDate: 2025-03-19
+author:
+  name: Mario Guerra
+  title: Senior Product Manager @ Microsoft
+authorAvatar: assets/img/authors/mario_guerra.png
+tags:
+  - release
+  - announcement
+---
+
+We're excited to announce the TypeSpec 1.0 Release Candidate, marking a significant milestone for the TypeSpec project. TypeSpec embodies a simple yet powerful principle: write your API once, and let TypeSpec take care of the rest. This straightforward approach allows you to focus on designing the perfect API contract while TypeSpec handles generating all the artifacts you need - from OpenAPI specifications to client libraries and server code scaffolding - saving you time and eliminating inconsistencies.
+
+## What is TypeSpec?
+
+TypeSpec is a language for describing API contracts. Using TypeSpec, you write your API definition in a concise, human-readable format, and TypeSpec takes care of generating:
+
+- API specifications (OpenAPI, JSON Schema, Protocol Buffers)
+- Server-side code skeletons (C#, JavaScript)
+- Client libraries in multiple languages (JavaScript, Python, C#, Java)
+- Documentation
+- Custom formats through our emitter framework
+
+This API-first development approach accelerates your initial API development by generating boilerplate code automatically, while also eliminating the need to maintain multiple, separate artifacts that can easily drift out of sync as your API evolves.
+
+## Who's using TypeSpec?
+
+TypeSpec is already being used by teams both inside and outside Microsoft to streamline their API development workflows:
+
+> "One of the highlights [of using TypeSpec] was our ability to author API specs in a very simple and intuitive language. This meant that even [our PMs] who are not engineers were able to understand these API specifications. They could review them, make comments, and participate in designing those APIs."
+>
+> — _MS Learn Team_
```

Comment:
I would spell out Microsoft, not use MS.  Do they know we'll be quoting them?

Reply:
They don't know, I'll inform them.